src/components/Tooltip/Tooltip.tsx

```diff
@@ -232,7 +232,7 @@ export function Tooltip<FCProps = DefaultTooltipProps>(
         {triggerElement}
         <span
           data-testid="tooltipBody"
-          title={label}
+          title={typeof label === 'string' ? label : undefined}
```

Comment:
@lpsinger Instead of silently assigning `label` to `undefined` without a way to override, what do you think about:

1. Add an optional `title` prop which is a `string`
2. If the `title` prop is passed in/has a value, that value is assigned to the span's `title`.
3. If `title` is not passed in, then use `label` if it is a string

Reply:
Sure thing. Done.